.github/pull_request_template.md

```diff
@@ -1,6 +1,18 @@
-## NOTICE
+<!--
+  Thanks for sending a pull request!
+    1. If this is your first time, please read our contributor guidelines: https://cwiki.apache.org/confluence/display/HADOOP/How+To+Contribute
+    2. Make sure your PR title starts with JIRA issue id, e.g., 'HADOOP-17799. Your PR title ...'.
+-->
+
+### Description of PR
+
+
+### How was this patch tested?
+
+
+### For code changes:
+
+- [ ] Object storage: have the integration tests been executed and the endpoint declared according to the connector-specific documentation?
```

Comment:
Shall we also add this point `PR title starts with JIRA issue id, e.g., 'HADOOP-17799. Your PR title ...'.`?

Reply:
probably restore the lines that were removed in this PR.

Reply:
Thank you. Added a checklist to make sure the PR title is in the specified format: 458ae01